packages/react-dom-bindings/src/client/ReactDOMInput.js

```diff
@@ -418,26 +417,28 @@ export function restoreControlledInputState(element: Element, props: Object) {
   }
 }
 
-// In Chrome, assigning defaultValue to certain input types triggers input validation.
-// For number inputs, the display value loses trailing decimal points. For email inputs,
-// Chrome raises "The specified value <x> is not a valid email address".
-//
-// Here we check to see if the defaultValue has actually changed, avoiding these problems
-// when the user is inputting text
-//
-// https://github.com/facebook/react/issues/7253
+// Here we check to see if the defaultValue has actually changed and update it
 export function setDefaultValue(
   node: HTMLInputElement,
   type: ?string,
-  value: ToStringValue,
+  value: mixed,
+  force: ?boolean,
 ) {
+  if (['button', 'file', 'hidden', 'image', 'reset', 'submit'].includes(type))
```

Comment:
We prefer simple strict equality checks and `{}` around for our if blocks. It's more familiar to us.

Reply:
✅